### ISSUE TITLE
Slope lines 

### ISSUE THREAD

**Issue by jenskastensson** [NicoRio42/cassini#10]
Terrain with many depressions (e.g karstic dolines), slope lines are important. Is it doable to add?

Here's an example:  [Plateau de Calern](https://www.oca.eu/fr/visites-individuels/decouvrez-le-site-du-plateau-de-calern)
![Cassini map dolines](https://github.com/user-attachments/assets/3982e2d7-17de-40c4-9aa5-f63340ce2fef)
![dolines-arial](https://github.com/user-attachments/assets/c7098e4d-1f46-4d49-aedc-3ec3fcc8c749)

[GoogleMaps](https://www.google.com/maps/@43.7488064,6.9284871,278a,35y,350.38h,64.64t/data=!3m1!1e3?entry=ttu&g_ep=EgoyMDI0MTEyNC4xIKXMDSoASAFQAw%3D%3D)

**Comment by NicoRio42** [comment]
Hello Jens, thanks for you're feedbacks!
Indeed, I planned to work on a solution for this. I am not satisfied with Karttapullautin's way of handling this (purple lines for depressions), I have to dig if it is possible to add slopes lines like you suggested.

**Comment by jenskastensson** [comment]
I agree that the solution with purple depression lines is not the best solution, but a may be ok as a temporary workaround?

**Comment by NicoRio42** [comment]
Yes it could be, i'll see when I work on this problem. Right now all my efforts are on building the distributed compute system for mapant.fr. I'll do a first try soon and then switch back to improving Cassini 🙂

**Comment by jenskastensson** [comment]
Purple lines tested, looks good! Will test IRL in a few weeks! Thanks!
